lib/gui/components/safe-webview/preload/safe-webview.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const electron = require('electron');
+
+/**
+ * @summary The SafeWebview functionality to be exposed to the guest
+ */
+window.safeWebview = {
+  sendToHost: (object) => {
```

Comment:
From a security point of view, I guess we need to be very careful what this IPC channel is used for?